app/views/layouts/_navigation.html.haml

```diff
@@ -13,17 +13,32 @@
         %ul.nav.navbar-nav#splash-nav
           = content_for :splash_nav
       -if user_signed_in?
-        %ul.nav.navbar-nav.navbar-right
-          %li.dropdown
-            %a.dropdown-toggle{"data-toggle" => "dropdown", :href => "#", id: "current-user-detail"}
-              - if not current_user.name.blank?
-                #{current_user.name}
-              -else
-                #{current_user.email}
-              = image_tag(current_user.gravatar_url(size: '18'), title: "Yo #{current_user.name}!", :alt => '')
-              %b.caret
-            %ul.dropdown-menu
-              = render 'layouts/user_menu'
+        .btn-group.pull-right
+          %ul.nav.navbar-nav.navbar-right
+            %li.dropdown
+              %a.dropdown-toggle{"data-toggle" => "dropdown", :href => "#", id: "current-user-detail"}
+                - if not current_user.name.blank?
+                  #{current_user.name}
+                -else
+                  #{current_user.email}
+                = image_tag(current_user.gravatar_url(size: '18'), title: "Yo #{current_user.name}!", :alt => '')
+                %b.caret
+              %ul.dropdown-menu
+                = render 'layouts/user_menu'
+          - if can? :manage, Conference
```

Comment:
if can? :manage Comment is more appropriate here, perfectly true. I'll update code